### PR TITLE
 [moonbase] Moonbase Strategy

### PR DESCRIPTION
**Pull request by RealWorldVesper**
Taking over for other Moonbase strategy submitted.  Please consider this the official strategy submission from Moonbase.markets.

Please let me know if the fix made to your previous request for limiting of moonbaseLPs is sufficient to alleviate the issue.

### CODE REVIEW

**Comment by RealWorldVesper** [comment]
Hi @ChaituVR 

taking over this process from @silverstar679 who is leaving the project.  his previous submission was here:

https://github.com/snapshot-labs/snapshot-strategies/pull/1272

I've replicated his final submission, and also updated with the last request you made regarding the limiting of moonbaseLPs.

Please advise if further edits are required.

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "@ethersproject/strings": "^5.6.1",
     "@ethersproject/units": "^5.6.1",
     "@ethersproject/wallet": "^5.6.2",
-    "@snapshot-labs/snapshot.js": "^0.5.8",
+    "@snapshot-labs/snapshot.js": "^0.5.5",
```
Comment (ChaituVR) [inline package.json:37]: Hi @RealWorldVesper Can revert this and yarn.lock plz :)

**Comment by RealWorldVesper** [comment]
Hi @ChaituVR this is now completed.  Please let me know if anything else is needed.

**Comment by ChaituVR** [comment]
Missing one thing @RealWorldVesper need to add this strategy to src/strategies/index.ts :) sorry for missing it earlier

File: src/strategies/moonbase/index.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    addresses.forEach((address: any) => {
+      multicall.call(
+        `lpUsers.${address}.${lp.pid}`,
+        options.masterChef,
+        'userInfo',
+        [lp.pid, address]
+      );
+    });
+  });
+  const [result] = await Promise.all([multicall.execute()]);
```
Comment (ChaituVR) [inline src/strategies/moonbase/index.ts:63]: No need for promise.all

**Review by ChaituVR — COMMENTED** [review]
Everything else looks good to me :)

**Comment by RealWorldVesper** [comment]
Hi @ChaituVR :)

Updates made inline with your guidance.  Please let me know if this is ok now.  Thanks

File: src/strategies/moonbase/index.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import examplesFile from './examples.json';
+
+export const author = 'moonbase';
```
Comment (ChaituVR) [inline src/strategies/moonbase/index.ts:6]:
```suggestion
export const author = 'MoonbaseMarkets';
```

**Comment by RealWorldVesper** [comment]
Confirmed thanks @ChaituVR

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**